src/database/inventory.js

```diff
@@ -0,0 +1,39 @@
+module.exports = (knex) => {
+  function findAllItemsFromPlayer(playerUuid) {
+    return knex('item')
+      .select('uuid', 'type', 'value', 'name')
+      .where({ owner: playerUuid })
+      .orderBy('value', 'name');
+  }
+
+  function getItemDetails(playerUuid, itemUuid) {
+    return knex('item')
+      .select('*')
+      .where({ owner: playerUuid })
+      .andWhere({ uuid: itemUuid })
+      .first();
+  }
+
+  async function sellItem(itemUuid) {
```

Comment:
This function executes two queries that make changes to the data, both need to be executed correctly, otherwise neither should run

To solve this you can use [knex Transactions](https://knexjs.org/guide/transactions.html)

Reply:
pushed a commit just now that should fix this, thanks for the insight